packages/strapi-plugin-upload/admin/src/translations/en.json

```diff
@@ -11,6 +11,7 @@
   "control-card.edit": "Edit",
   "control-card.replace-media": "Replace Media",
   "control-card.save": "Save",
+  "filter.name": "Filters",
```

Comment:
This is not needed the translation exists in the `strapi-admin` package.  

> In fact, when a translation id is not prefixed by the `pluginId` (which `getTrad`)  returns the administration panel will look for the translation in the `strapi-admin` one. With this way, we can (try) to maintain less translation and reuse them as much as possible.

Reply:
ok, I refactored it. thanks for your review.